engines/bastion_katello/lib/bastion_katello/engine.rb

```diff
@@ -43,7 +43,6 @@ class Engine < ::Rails::Engine
             'katelloAgentPresent' => ::Katello.with_katello_agent?,
             'remoteExecutionPresent' => ::Katello.with_remote_execution?,
             'hostToolingEnabled' => (::Katello.with_katello_agent? || ::Katello.with_remote_execution?) ? true : false,
-            'remoteExecutionByDefault' => ::Katello.remote_execution_by_default?
```

Comment:
`remoteExecutionByDefault` is used by angular code. Maybe this line should be removed by the PR that cleans up angular code.

For this one, we can keep it as:
```
'remoteExecutionByDefault' => true
```

Reply:
@chris1984 If you can move this change to your PR, I'll remove it here.

Reply:
:+1:

Reply:
actually, If I remove this line but keep the method removed, it will break all the tests on this PR. @lfu I plan to merge the Angular PR first. I'm thinking it may be easier to just keep this PR the way it is..   @chris1984

Reply:
Lol that's fine, I'll take it out of mine